lfview/resources/spatial/options.py

```diff
@@ -193,6 +193,18 @@ class _BaseElementOptions(_BaseOptions):
 class OptionsPoints(_BaseElementOptions):
     """PointSet visualization options"""
 
+    size = properties.Instance(
+        'Default point size on the element',
+        OptionsSize,
+        default=OptionsSize,
+    )
+    shape = properties.StringChoice(
+        'Points are displayed as squares or spheres',
+        default='Square',
+        choices=['Square', 'Sphere'],
+        required=False,
```

Comment:
Not sure we want `required=False` - Having a default means it may be unspecified, but if we leave it as optional, that means we (i.e. View / other clients) have to deal with the case where someone explicitly removes it.